### PR TITLE
KVM / Xen test fixes

### PR DESCRIPTION
**Pull request by cbosdo**
## What does this PR change?

Fix test breaks introduced by https://github.com/uyuni-project/uyuni/pull/3986

## GUI diff

No difference.

- [X] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes
- [X] **DONE**

## Test coverage
- No tests: already covered

- [X] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [X] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: testsuite/features/secondary/minkvm_guests.feature [hunk 1]
```diff
@@ -21,8 +21,10 @@ Feature: Be able to manage KVM virtual machines via the GUI
     And I wait until I see "Successfully bootstrapped host!" text
     And I wait until onboarding is completed for "kvm_server"
 
-  Scenario: Setting the virtualization entitlement for KVM
+  Scenario: Showing the KVM host system overview
```
Comment (Bischoff) [inline testsuite/features/secondary/minkvm_guests.feature:24]: Show

File: testsuite/features/secondary/minkvm_guests.feature [hunk 2]
```diff
@@ -21,8 +21,10 @@ Feature: Be able to manage KVM virtual machines via the GUI
...
     Given I am on the Systems overview page of this "kvm_server"
+
+  Scenario: Setting the virtualization entitlement for KVM
```
Comment (Bischoff) [inline testsuite/features/secondary/minkvm_guests.feature:27]: Set

File: testsuite/features/secondary/minxen_guests.feature [hunk 3]
```diff
@@ -21,8 +21,10 @@ Feature: Be able to manage XEN virtual machines via the GUI
     And I wait at most 300 seconds until I see "Successfully bootstrapped host!" text
     And I wait until onboarding is completed for "xen_server"
 
-  Scenario: Setting the virtualization entitlement for Xen
+  Scenario: Showing the Xen host system overview
```
Comment (Bischoff) [inline testsuite/features/secondary/minxen_guests.feature:24]: Show

File: testsuite/features/secondary/minxen_guests.feature [hunk 4]
```diff
@@ -21,8 +21,10 @@ Feature: Be able to manage XEN virtual machines via the GUI
...
     Given I am on the Systems overview page of this "xen_server"
+
+  Scenario: Setting the virtualization entitlement for Xen
```
Comment (Bischoff) [inline testsuite/features/secondary/minxen_guests.feature:27]: Set

**Review by Bischoff — CHANGES_REQUESTED** [review]
lgtm - nitpicked

File: testsuite/features/secondary/minkvm_guests.feature (hunk 1 above)
Comment (cbosdo) [reply testsuite/features/secondary/minkvm_guests.feature]: fixed

File: testsuite/features/secondary/minxen_guests.feature (hunk 3 above)
Comment (cbosdo) [reply testsuite/features/secondary/minxen_guests.feature]: Fixed

File: testsuite/features/secondary/minxen_guests.feature (hunk 4 above)
Comment (cbosdo) [reply testsuite/features/secondary/minxen_guests.feature]: Fixed

File: testsuite/features/secondary/minkvm_guests.feature (hunk 2 above)
Comment (cbosdo) [reply testsuite/features/secondary/minkvm_guests.feature]: fixed

**Pull request merged**